### PR TITLE
Stats: remove Stats NUX modules

### PR DESCRIPTION
**Pull request by jonathansadowski**
This PR removes the Stats NUX component. This was originally a test that performed no better than the control. There have since been several issues reported with the logic for displaying Stats NUX resulting in false positives, and denying users who should be able to view their Stats the ability to view them.

Before:

image

After:

image

To test: create a new WordPress and make sure that you're immediately able to view your Stats.

### CODE REVIEW

File: client/my-sites/stats/controller.js [hunk 1]
```diff
@@ -346,9 +327,7 @@ module.exports = {
 			baseAnalyticsPath,
 			chartQuantity = 10,
 			siteComponent,
-			twoWeeksAgo = i18n.moment().subtract( 2, 'week' ),
-			siteCreated,
-			isNux;
+			twoWeeksAgo = i18n.moment().subtract( 2, 'week' );
```
Comment (timmyc) [inline client/my-sites/stats/controller.js]: This can also be removed since it is no longer used.

File: client/my-sites/stats/controller.js [hunk 2]
```diff
@@ -139,9 +138,7 @@ module.exports = {
 			momentSiteZone = i18n.moment(),
 			StatsComponent = Insights,
 			twoWeeksAgo = i18n.moment().subtract( 2, 'week' ),
-			siteCreated,
-			postCount,
-			isNux;
+			postCount;
```
Comment (timmyc) [inline client/my-sites/stats/controller.js]: `postCount` and `twoWeeksAgo` can also be removed here.

**Comment by timmyc** [comment]
Thanks again for cleaning this up @jonathansadowski - looks good and tests out well.  :shipit:

**Comment by jonathansadowski** [comment]
Thanks for the review @timmyc

**Pull request merged**